src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/monitor-services/monitor-services-running-kubernetes.mdx

```diff
@@ -148,50 +151,13 @@ newrelic-infrastructure:
 ```
 
 <Callout variant="tip">
-  Notice we are specifying `--tls --port 10250`. Earlier versions of the integration might have worked without this, as of [version 3](/docs/kubernetes-pixie/kubernetes-integration/get-started/changes-since-v3) of the Kubernetes integration it is mandatory to specify them. The reason for this change is that the integration now connects to the Kubelet using the nodeIP rather than `localhost`, with the former requiring TLS while the latter did not.
+  Notice we are specifying `--tls --port 10250`. Earlier versions of the integration might have worked without this, as of [version 3](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3) of the Kubernetes integration it is mandatory to specify them. The reason for this change is that the integration now connects to the Kubelet using the nodeIP rather than `localhost`, with the former requiring TLS while the latter did not.
 </Callout>
 
 Integrations targeting other pods should have their own section next to `redis-sampleapp.yaml`.
 
 Integrations are standalone binaries and are executed by the infrastructure agent that's included in the `newrelic-nrk8s-kubelet-xxxxx` pods. Config files are deployed to all pods of the `nrk8s-kubelet` DaemonSet, but discovery ensures that each pod will only target service pods that're scheduled in the same node as that particular `nrk8s-kubelet` pod. If an instance of the `nrk8s-kubelet` DaemonSet does not have any pods matching the specified labels, the integration will not be executed by that ### instance.
```

Comment:
```suggestion
Integrations are standalone binaries and are executed by the infrastructure agent that's included in the `newrelic-nrk8s-kubelet-xxxxx` pods. Config files are deployed to all pods of the `nrk8s-kubelet` DaemonSet, but discovery ensures that each pod will only target service pods that're scheduled in the same node as that particular `nrk8s-kubelet` pod. If an instance of the `nrk8s-kubelet` DaemonSet does not have any pods matching the specified labels, the integration will not be executed by that `###` instance.
```

I'm not sure about what ### is referencing, but if it's something code, put it in code blocks